### DISCUSSION TITLE
Honor the ability for apps to bypass VPN when in VPN lockdown mode?

### DISCUSSION THREAD

**Discussion by realbiz21** [schwabe/ics-openvpn#1528] [Configuration problems]
In the Android settings, I have ics-openvpn configured as Always On and Block Connections Without VPN (lockdown mode).

However, there are some apps that I want to have access to my LAN. Therefore I check 'Allow apps to bypass' in the openvpn profile and select the apps. I also check the setting to enable LAN addresses to avoid the VPN.

It doesn't appear possible to do this while in VPN lockdown mode (Block connections enabled), but it works when not in lockdown.

Is it possible for ics-openvpn to honor the Bypass Apps setting while in VPN lockdown mode?

**Comment by schwabe** [comment]
Probably not, the option is just a flag the app sets to Android.
